code/controllers/subsystem/job.dm

```diff
@@ -49,7 +49,10 @@ SUBSYSTEM_DEF(job)
 		if(!job.map_check())
 			continue
 		occupations += job
-		name_occupations[job.title] = job
+		var/job_name = job.title
+		if(job.faction == FACTION_TERRAGOV_REBEL)
+			job_name += " Rebel"
+		name_occupations[job_name] = job
```

Comment:
why though just name it properly

Reply:
So admins can give it from VV.
It was mostly usefull from testing, but this might be usefull in game too

Reply:
give what from vv this is job names